### PR TITLE
fix(chat): add textarea with auto adjustable height

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

textarea with auto adjustable height

Issues:

- Issue #

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453827)

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453828)

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453833)

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453836)

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/453863)

File: apps/chat/src/components/Chat/ChatMessage/AdjustedTextarea.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  return (
+    <>
+      <textarea
+        {...props}
+        ref={hiddenTextareaRef}
+        name="hidden"
+        className={classNames(
+          className,
+          'absolute left-[-10000px] top-[-10000px]', // hidden, but exists
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/ChatMessage/AdjustedTextarea.tsx]:
```suggestion
          'invisible absolute', // hidden, but exists
```

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-970.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-970.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat-e2e/src/ui/selectors/chatSelectors.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export const ChatSelectors = {
   expandGroupEntity: '[data-qa="expand-group-entity"]',
   systemPrompt: '[data-qa="system-prompt"]',
   message: '[data-qa="message"]',
+  textarea: '[data-qa="chat-textarea"]',
```
Comment (irinakartun) [inline apps/chat-e2e/src/ui/selectors/chatSelectors.ts:19]: please, use more meaningful name, like requestTextarea

**Review by irinakartun — DISMISSED** [review]

**Review by irinakartun — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-970.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-970.nightly-test.deltixhub.io
>E2E tests status: success

**Review by irinakartun — APPROVED** [review]

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**